practice/004/answer/test/dune

```diff
@@ -0,0 +1,5 @@
+(include_subdirs no)
+
+(test
+ (name run)
+ (libraries ounit2 ex))
```

Comment:
```suggestion
(test (name run) (libraries ounit2 ex))
```

Reply:
Hi @cuihtlauac please help, I effected these changes and tried to run dune test --root . answer 
now am getting this error
dune test --root . answer 
File "dune", line 2, characters 12-15:
2 | (test (name run) (libraries ounit2 ex))
                ^^^
Error: Module "Run" doesn't exist.
File "answer/test/dune", line 2, characters 35-37:
2 | (test (name run) (libraries ounit2 ex))
                                       ^^
Error: Library "ex" not found.
-> required by _build/default/answer/test/run.exe
-> required by alias answer/test/runtest in answer/test/dune:2
did I miss something?

Reply:
Theses commands are working fine on my side with your PR. In which folder are you running them? Have you completed the setup of ocaml.org?